main/src/com/google/refine/commands/GetAllPreferencesCommand.java

```diff
@@ -63,6 +68,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
         
         for (String key : ps.getKeys()) {
             Object pref = ps.get(key);
+            // This skips prefs of type TopList. Not sure if that's intentional or an oversight
```

Comment:
Maybe it's normal, so that the user only sees the first level, that are returned by this call. Internal function that uses the preference mechanism must always be hidden by being TopList…

Maybe I'm wrong.

Regards, A.